app/src/main/java/io/keen/client/android/example/MyActivity.java

```diff
@@ -56,16 +61,63 @@ protected void onPause() {
     }
 
     public void handleClick(View view) {
+        Log.i(this.getLocalClassName(), "handling click on view " + view.getId());
         switch (view.getId()) {
             case R.id.send_event_button:
                 // Create an event to upload to Keen.
-                Map<String, Object> event = new HashMap<String, Object>();
+                Map<String, Object> event = new HashMap<>();
                 event.put("click-number", clickNumber++);
 
                 // Add it to the "purchases" collection in your Keen Project.
-                KeenClient.client().queueEvent("android-sample-button-clicks", event);
+                KeenClient.client().queueEvent(getCollection(), event);
+                break;
```

Comment:
functional programming is seeping its way into my brain - this `break` made me grimace